pkgs/applications/networking/cluster/kubectl/default.nix

```diff
@@ -1,7 +1,9 @@
 { stdenv, kubernetes, installShellFiles }:
-
+let
+  kubernetesKubectlOnly = kubernetes.override { components = []; };
+in
 stdenv.mkDerivation {
-  name = "kubectl-${kubernetes.version}";
+  name = "kubectl-${kubernetesKubectlOnly.version}";
```

Comment:
could you use pname/version instead ?

Reply:
do you mean change this to `name = "kubectl-${kubernetesKubectlOnly.pname}/${kubernetesKubectlOnly.version}";`?

Reply:
nope: just `pname = "kubectl"; version =  kubernetesKubectlOnly.version`